src/main/java/educonnect/model/student/TelegramHandle.java

```diff
@@ -10,8 +10,9 @@
 public class TelegramHandle {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Telegram handles should start with @";
-    public static final String VALIDATION_REGEX = "^@.*";
+            "Telegram handles should start with '@' and "
+                    + "contains at least 1 alphanumeric characters and/or underscores after it.";
+    public static final String VALIDATION_REGEX = "^@[a-zA-Z0-9_]+$";
```

Comment:
```suggestion
    public static final String VALIDATION_REGEX = "^@\\w+$";
```
A more concise way of achieving the same thing, `\w` regex is short form for `[a-zA-Z0-9_]`.
Explicitly showing a-zA-Z may be better for people less familiar with regex.

Reply:
agreed, I had to google \w hahaha